cmd/oras/attach.go

```diff
@@ -44,7 +44,7 @@ type attachOptions struct {
 func attachCmd() *cobra.Command {
 	var opts attachOptions
 	cmd := &cobra.Command{
-		Use:   "attach name<:tag|@digest> [file[:type]...]",
+		Use:   "attach [flags] <name>{:<tag>|@<digest>} [<file>[:<type>]...]",
```

Comment:
```suggestion
		Use:   "attach [flags] <name>{:tag|@digest} [file[:type]] ...",
```

Reply:
According to the syntax [guideline](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/command-line-syntax-key), we should not, e.g. https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd